css/css-fonts/font-face-local-ref.html

```diff
@@ -0,0 +1,7 @@
+<style>
```

Comment:
I would still include `<!doctype html>` here so that you're not in quirks mode. I don't know if will affect this test, but it's best to not have it as a suspected cause of failure when this test indeed fails :)

Reply:
ty! i got chop happy apparently!